### ISSUE TITLE
Minplayer does not resize

### ISSUE THREAD

**Issue by rhwendt** [ytmdesktop/ytmdesktop#223]
#### Problem 
Miniplayer does not resize. Maybe i'm doing something wrong but this doesn't seem like expected behavior.
###### (Describe the problem here) 
When I set the miniplayer to any of the size options in settings it never changes size
#### Environment 
 * YTMDesktop: - 
 * Platform: linux 
 * Arch: x64 
 * Version: 5.4.0-37-generic 
 * Memory: 16 GB 
 * Node: 12.14.1 

OS: Ubuntu 20.04 LTS x86_64

#### Prints 
These are full screen prints of my secondary monitor 1920 x 1080
![settings](https://user-images.githubusercontent.com/42274154/86176246-acab9780-baf2-11ea-9a7d-ce89edca7562.png)
![miniplayer](https://user-images.githubusercontent.com/42274154/86176241-ab7a6a80-baf2-11ea-941d-22ab1dc3c1bc.png)

###### (if possible)

**Comment by rickinbsb** [comment]
Hey there, thanks for reporting this!
Not only that, but the miniplayer is supposed to be always 1:1

**Comment by rhwendt** [comment]
For what its worth I don't see this issue on the Windows release.

**Comment by rickinbsb** [comment]
I'll see if I can reproduce it.
Are you on gnome? If yes, can you check if some extension might be messing with the window?

**Comment by rhwendt** [comment]
Yeah I have gnome installed.

```
❯ gnome-extensions list
desktop-icons@csoriano
ubuntu-appindicators@ubuntu.com
ubuntu-dock@ubuntu.com
```

I disabled the desktop-icons extension and the issue persists.

**Comment by rickinbsb** [comment]
right, thank you

**Comment by evilbenfranklin** [comment]
This doesn't seem to be DE-centric; I'm having the same issue with a fresh snap install on Manjaro KDE Plasma spin.

**Comment by rickinbsb** [comment]
Fixed on v1.12, thank you!
